src/wavm.cpp

```diff
@@ -215,6 +215,16 @@ ExecutionResult WavmEngine::execute(
   }
 }
 
+struct WavmInterfaceKeeper
```

Comment:
Can you move this into the `wavm_host_module` namespace? (Just after where `interface` is declared)